src/main/java/io/vertx/core/net/TCPSSLOptions.java

```diff
@@ -56,6 +57,11 @@
    */
   public static final int DEFAULT_IDLE_TIMEOUT = 0;
 
+  /**
+   * Default idle time unit = SECONDS (backward compatibility)
```

Comment:
please remove the "backward compatibility", because if not, I don't see why we would specify a different time unit.

Reply:
Done.

P.S should I squash all commits into single final one ?